.github/workflows/verify.yml

```diff
@@ -37,11 +37,7 @@ jobs:
 
       - name: docker-compose build
         run: |
-          curl -L https://github.com/docker/compose/releases/download/1.22.0/docker-compose-`uname -s`-`uname -m` > docker-compose
-          chmod +x docker-compose
-          sudo mv docker-compose /usr/bin
-
-          /usr/bin/docker-compose build
+          docker compose build
```

Comment:
The standalone `docker-compose` tool is baked into `docker` nowdays, so no need to use this pattern